app/lib/clients/create_employees.rb

```diff
@@ -0,0 +1,54 @@
+class Clients::CreateEmployees
+  def initialize(employees, time_off)
+    @employees = employees
+    @time_off = time_off
+    @dates = create_dates
+  end
+
+  def create_dates
+    day_before_yesterday = (Time.zone.yesterday - 1).to_formatted_s(:iso8601)
+    yesterday = Time.zone.yesterday.to_formatted_s(:iso8601)
+    today = Time.zone.now.to_formatted_s(:iso8601)
+    tomorrow = Time.zone.tomorrow.to_formatted_s(:iso8601)
+    day_afer_tomorrow = (Time.zone.tomorrow + 1).to_formatted_s(:iso8601)
+    [day_before_yesterday, yesterday, today, tomorrow, day_afer_tomorrow]
+  end
+
+  def find_birthday_employees
+    birthdays = []
+    @employees.each do |employee|
+      if employee['birthday'].present? &&
+         @dates.any? { |d|
+           d.include? employee['birthday']
+         }
+        birthdays.append(employee)
+      end
+    end
+    birthdays
+  end
+
+  def find_anniversary_employees
+    anniversaries = []
+    @employees.each do |employee|
+      if employee['hireDate'].present? &&
+         @dates.any? { |d|
+           d.include? employee['hireDate']
+         }
```

Comment:
We repeat this `@dates.any?` block. Could we extract and name it something like `dates_contains(...)` or `this_week_contains(...)`?